store/tikv/region_cache.go

```diff
@@ -347,9 +348,11 @@ func (c *RegionCache) GetTiFlashRPCContext(bo *Backoffer, id RegionVerID) (*RPCC
 
 	regionStore := cachedRegion.getStore()
 
-	tikvCnt := 0
-	for i, store := range regionStore.stores {
-		addr, err := c.getStoreAddr(bo, cachedRegion, store, i)
+	tikvCnt, sIdx := 0, rand.Int()%len(regionStore.stores)
```

Comment:
https://mozillazg.com/2019/05/go-be-careful-of-math-rand-functions.html maybe this method will be called frequently and better not use global rand